### DISCUSSION TITLE
SE_HW1_14022_Q1

### DISCUSSION THREAD

**Discussion by AmooHashem** [ssc-public/Software-Engineering#1] [Q&A]
سوالات خود پیرامون سوال اول از تمرین اول درس - ترم ۱۴۰۲۲ را اینجا بپرسید!

**Comment by hamriouz** [comment]
توی این سوال برخی از سبک‌های رهبری نام برده شدن و برای بقیه سه نقطه گذاشته شده. این یعنی ما باید همه‌ی سبک‌های رهبری که توی منبع ذکر شدن رو مورد بررسی قرار بدیم یا فقط مواردی که توی صورت سوال امدن؟

**Reply by mahdimostajer** [reply]
سلام تمام سبک‌های رهبری که در کتاب ذکر شده رو بررسی کنید. فقط بررسی Authentic ضرورتی ندارد چون همانطور که در کتاب اشاره شده است Authentic بیشتر از اینکه سبک رهبری باشد، صفت و ویژگی رهبری محسوب می‌شود.
